src/System.Management.Automation/engine/lang/parserutils.cs

```diff
@@ -1478,7 +1478,13 @@ internal int UpperBound
         }
 
         private int _current;
-        public object Current
+
+        Object IEnumerator.Current
```

Comment:
This should swap places with `private int _current;` so that the field can be closed to the property it backs.